src/deep-link-to-native-app.ts

```diff
@@ -22,13 +22,13 @@ export class NativeAppLauncher {
       appendQueryParameter
     };
 
+    // TODO: is this true?
     // Let all the page render finish.
```

Comment:
do we prefer to handle this outside the library? or do we want to implement our domready? 🏄